azurelinuxagent/common/logcollector.py

```diff
@@ -45,6 +45,7 @@
 
 CGROUPS_UNIT = "collect-logs.scope"
 
+OOM_KILLED_ERRCODE = -9
```

Comment:
I would not make this specific to OOM... -9 it is just a generic force kill. let's rename this and add comments that it being used to attempt to capture the termination by OOM

Reply:
Updated. I added the comments where this value is used, thought that fit better.